recipes/gelidum/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "gelidum" %}
+{% set version = "0.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 67ddde79967a39799414009710edc0b306e5b210e7ec835739f5172eb0498f31
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.7.0
```

Since we need to change above, we usually just use:
```suggestion
    - python >=3.7
```